st2common/st2common/validators/api/reactor.py

```diff
@@ -70,4 +74,12 @@ def validate_trigger_parameters(trigger_type_ref, parameters):
     cleaned = util_schema.validate(instance=parameters, schema=parameters_schema,
                                    cls=util_schema.CustomValidator, use_default=True,
                                    allow_default_none=True)
+
+    # Additional validation fron CronTimer
+    if trigger_type_ref == ResourceReference.to_string_reference(SYSTEM_PACK_NAME, 'st2.CronTimer'):
```

Comment:
Do you really have to validate in common code and look for a special trigger_type_ref?